### PR TITLE
Chromium Chronicle 23

### PR DESCRIPTION
**Pull request by rachelandrew**
Adds the Chromium Chronicle 23.

### CODE REVIEW

File: site/en/blog/chromium-chronicle-23/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Binary Authorization is implemented as a two-step system.
+The system generates a [provenance](https://csrc.nist.gov/glossary/term/Provenance)
+with build time information;
+the enforcement of policy occurs before signing or installing software.
+
+```python
+def CreateProvenance(self, build_manifest: Mapping[str, Any]):
+"""Builder generates and signs provenance given build manifest. Signed JWT is placed
+  alongside built artifact."""
```
Comment (petele) [inline site/en/blog/chromium-chronicle-23/index.md:52]:
```suggestion
  """Builder generates and signs provenance given build manifest. Signed JWT is placed
  alongside built artifact."""
```

File: site/en/blog/chromium-chronicle-23/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+For Chrome, before signing the produced software artifacts using Google's signing infrastructure,
+the policy is enforced to meet particular minimum security requirements of the build.
+
+```python
+def VerifyProvenance(self, artifact_hash: str, provenance: str):
+"""Provenance is verified using a policy engine service before signing an artifact."""
```
Comment (petele) [inline site/en/blog/chromium-chronicle-23/index.md:60]:
```suggestion
  """Provenance is verified using a policy engine service before signing an artifact."""
```

**Review by petele — APPROVED** [review]
Two minor formatting suggestions. Looks great, thanks @rachelandrew

**Comment by petele** [comment]
Publish July 28th

File: site/en/blog/chromium-chronicle-23/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+title: "The Chromium Chronicle #23: Verified builds in Chrome Infra"
+description: >
+  Implementing a policy enforcement check for Chrome and the infrastructure
+  has enabled us to verify that certain minimum standards for security are met.
+layout: 'layouts/blog-post.njk'
+date: 2021-07-21
```
Comment (petele) [inline site/en/blog/chromium-chronicle-23/index.md:7]:
```suggestion
date: 2021-07-28
```

So it's date matches the publication date.

File: site/en/blog/chromium-chronicle-23/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+def VerifyProvenance(self, artifact_hash: str, provenance: str):
+  """Provenance is verified using a policy engine service before signing an artifact."""
+```
+
+Requirements are divided into roughly 4 areas:
+
+- _Source control:_ protects the data that went into the build.
+- _Build:_ protects the process that converts the source into binary.
+- _Provenance:_ attestation that contains verifiable build manifest.
+- _Policy:_ rules that determine if a given artifact qualifies in a given context.
```
Comment (rachelandrew) [inline site/en/blog/chromium-chronicle-23/index.md:68]:
```suggestion
- *Source control:* protects the data that went into the build.
- *Build:* protects the process that converts the source into binary.
- *Provenance:* attestation that contains verifiable build manifest.
- *Policy:* rules that determine if a given artifact qualifies in a given context.
```

File: site/en/blog/chromium-chronicle-23/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+def VerifyProvenance(self, artifact_hash: str, provenance: str):
+  """Provenance is verified using a policy engine service before signing an artifact."""
+```
+
+Requirements are divided into roughly 4 areas:
+
+- *Source control:* protects the data that went into the build.
+- *Build:* protects the process that converts the source into binary.
+- *Provenance:* attestation that contains verifiable build manifest.
+- *Policy:* rules that determine if a given artifact qualifies in a given context.
```
Comment (rachelandrew) [inline site/en/blog/chromium-chronicle-23/index.md:68]:
```suggestion
- **Source control:** protects the data that went into the build.
- **Build:** protects the process that converts the source into binary.
- **Provenance:** attestation that contains verifiable build manifest.
- **Policy:** rules that determine if a given artifact qualifies in a given context.
```

**Pull request merged**